platform/darwin/src/MGLStyle.mm

```diff
@@ -625,12 +627,26 @@ - (void)localizeLabelsIntoLocale:(nullable NSLocale *)locale {
     }];
 }
 
+- (void)setAccessiblePlaceSourceIdentifiers:(NSSet *)accessiblePlaceSourceLayerIdentifiers {
+    if (![_accessiblePlaceSourceIdentifiers isEqualToSet:accessiblePlaceSourceLayerIdentifiers]) {
+        _accessiblePlaceSourceIdentifiers = accessiblePlaceSourceLayerIdentifiers;
+    }
+}
+
+- (NSSet *)accessiblePlaceSourceIdentifiers {
+    return _accessiblePlaceSourceIdentifiers;
+}
+
 - (NSArray<MGLStyleLayer *> *)placeStyleLayers {
     NSSet *streetsSourceIdentifiers = [self.mapboxStreetsSources valueForKey:@"identifier"];
-    
+
     NSSet *placeSourceLayerIdentifiers = [NSSet setWithObjects:@"marine_label", @"country_label", @"state_label", @"place_label", @"water_label", @"poi_label", @"rail_station_label", @"mountain_peak_label", @"natural_label", @"transit_stop_label", nil];
+    if (self.accessiblePlaceSourceIdentifiers == nil) {
+        _accessiblePlaceSourceIdentifiers = [NSMutableSet set];
+    }
+
     NSPredicate *isPlacePredicate = [NSPredicate predicateWithBlock:^BOOL (MGLVectorStyleLayer * _Nullable layer, NSDictionary<NSString *, id> * _Nullable bindings) {
-        return [layer isKindOfClass:[MGLVectorStyleLayer class]] && [streetsSourceIdentifiers containsObject:layer.sourceIdentifier] && [placeSourceLayerIdentifiers containsObject:layer.sourceLayerIdentifier];
+        return [layer isKindOfClass:[MGLVectorStyleLayer class]] && (([streetsSourceIdentifiers containsObject:layer.sourceIdentifier] && [placeSourceLayerIdentifiers containsObject:layer.sourceLayerIdentifier]) || [self.accessiblePlaceSourceIdentifiers containsObject:layer.sourceIdentifier]);
```

Comment:
This method returns an array of identifiers of layers within sources used by layers, not an array of identifiers of sources used by layers. For example, it returns `country_label`, not `composite`.